xml/ldap_searchresult.xml

```diff
@@ -0,0 +1,735 @@
+    <description>Oracle Virtual Directory</description>
+    <example service.version="11.1.1.6.0" _encoding="base64">
+      b3JjbGRpcmVjdG9yeXZlcnNpb24xEAQOT1ZEIDExLjEuMS42LjAwLgQSc3VwcG9ydGVkRQ==
+    </example>
+    <param pos="0" name="service.vendor" value="Oracle"/>
+    <param pos="0" name="service.product" value="Virtual Directory Server"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+
+  <fingerprint pattern="metaProductID.*\x04\vmetaVersion1\r\x04.([\d.]+)0.\x04">
+    <description>estos MetaDirectory</description>
+    <example service.version="3.5.22.4291" _encoding="base64">
+      BA1tZXRhUHJvZHVjdElEMQYEBDExMDIwHAQLbWV0YVZlcnNpb24xDQQLMy41LjIyLjQyOTEwF
+      gQ=
+    </example>
+    <param pos="0" name="service.vendor" value="estos"/>
+    <param pos="0" name="service.product" value="MetaDirectory Server"/>
+    <param pos="1" name="service.version"/>
+    <param pos="0" name="os.vendor" value="Microsoft"/>
+    <param pos="0" name="os.family" value="Windows"/>
+  </fingerprint>
+
+  <fingerprint pattern="dsaVersion1.\x04,DC Directory Server v(\d+\.\d+[\d.]* \([\w. ]+\))0.\x04">
+    <description>Cisco Data Connection Directory</description>
+    <example service.version="8.1.00 (build 20150305)" _encoding="base64">
+      ZHNhVmVyc2lvbjEuBCxEQyBEaXJlY3RvcnkgU2VydmVyIHY4LjEuMDAgKGJ1aWxkIDIwMTUwM
+      zA1KTBeBAs=
+    </example>
+    <param pos="0" name="service.vendor" value="Cisco"/>
+    <param pos="0" name="service.product" value="Data Connection Directory"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+
+  <!-- Unbound -->
+  <fingerprint pattern="vendorName1.\x04.UnboundID Corp.0.\x04\rvendorVersion1.\x04.UnboundID Directory Server ([\d.]+)0\f">
+    <description>UnboundID Directory Server</description>
+    <example service.version="5.1.5.2" _encoding="base64">
+      dmVuZG9yTmFtZTERBA9VbmJvdW5kSUQgQ29ycC4wNQQNdmVuZG9yVmVyc2lvbjEkBCJVbmJvd
+      W5kSUQgRGlyZWN0b3J5IFNlcnZlciA1LjEuNS4yMAw=
+    </example>
+    <param pos="0" name="service.vendor" value="UnboundID"/>
+    <param pos="0" name="service.product" value="UnboundID Directory Server"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+
+  <fingerprint pattern="vendorName1.\x04.UnboundID Corp.0.\x04\rvendorVersion1.\x04.UnboundID Directory Proxy Server ([\d.]+)0\f">
+    <description>UnboundID Directory Proxy Server</description>
+    <example service.version="4.7.0.7" _encoding="base64">
+      dmVuZG9yTmFtZTERBA9VbmJvdW5kSUQgQ29ycC4wOwQNdmVuZG9yVmVyc2lvbjEqBChVbmJvd
+      W5kSUQgRGlyZWN0b3J5IFByb3h5IFNlcnZlciA0LjcuMC43MAw=
+    </example>
+    <param pos="0" name="service.vendor" value="UnboundID"/>
+    <param pos="0" name="service.product" value="UnboundID Directory Proxy Server"/>
+    <param pos="1" name="service.version"/>
+  </fingerprint>
+
+  <fingerprint pattern="namingContexts1.\x04.cn=.?[Pp][Bb][Xx].*\x04.ldapServiceName1.\x04.IPVA-\w+-" flags="REG_MULTILINE">
```

Comment:
This hasn't happened yet, but #75 aims to remove the `flags` attribute since the constants we specify there are language specific and don't necessarily translate to something in, say, Java.  ~Lately we prefer that the regular expression be written using the non-grouping modifiers instead -- `(?m:<your regex>)`